checks.d/nginx.py

```diff
@@ -93,7 +93,7 @@ def parse_text(cls, raw, tags):
             handled = int(parsed.group(2))
             requests = int(parsed.group(3))
             output.extend([('nginx.net.conn_opened_per_s', conn, tags, 'rate'),
-                           ('nginx.net.conn_handled_per_s', conn, tags, 'rate'),
+                           ('nginx.net.conn_dropped_per_s', conn-handled, tags, 'rate'),
```

Comment:
Thanks a lot for this @kky !

Tiny nitpick could you add a whitespace on each side of the - sign to respect PEP8 code style please ?

Soit would be:
```python
('nginx.net.conn_dropped_per_s', conn - handled, tags, 'rate'),
```

Thanks again!